internal/testing/testing.go

```diff
@@ -12,24 +12,20 @@
 //    License for the specific language governing permissions and limitations
 //    under the License.
 
-//go:build testing
-// +build testing
```

Comment:
What is the reason for these build constraints?

`_test.go` files are not part of go build - therefore don't go into production code - and dependencies of these files also do not get included? Is there any other reason I miss?

Reply:
This package holds common functions for testing only. These functions can be used in tests in any other package, while any code you define in _test.go are scoped only to the package the tests are in. You cannot import them in tests defined in other packages.

This build constraint makes these functions available only during testing. We don't want to build them otherwise, and suffixing them with _test.go would make them inaccessible to other packages.

You can run:

```bash
make test
```

which sets this tag and runs the tests. If ```vscode``` is complaining, you can set:

```json
"go.buildTags": "testing"
``` 

in your ```settings.json```

Reply:
Ok... I will drop the commit.